### PR TITLE
Initiate a page for upspeak

### LINKED ISSUE #965: Implement a new webpage for UpSpeak
**Is your feature request related to a problem?**
- We need to create a new webpage for the Upspeak program

**Describe the solution you'd like**
- Create a new page using the following content.

**Description :** 
UpSpeak
Over the last few years of delivering SEF-initiated programs, we noticed one of the biggest challenges faced by undergraduate students was their ability to speak in English. While their reading and writing skills were very good, their level of spoken English was not on par. These were due to a number of reasons including,g inadequate resources to practice spoken English in a safe environment and focus on mastering academic English using rigid patterns not transferrable when speaking
In response to this, UpSpeak was introduced by SEF during June 2021. This is a free 12-week program aimed at creating a non-competitive safe environment for undergraduates in Sri Lanka whose second language is English and are looking to develop the following skills as published by Cambridge Papers in ELT:
- Develop their conversational competence,
- Become more proficient in the use of appropriate, rather than just accurate language,
- Learn to play with language and take risks with it,
- Building positive group dynamics for learning

**FAQ section**
- How do I apply for the program?
   - Our first cohort kicked off during June 2021. Keep an eye out on our social media sites for future cohort intakes.
- How do I become a facilitator?
   - Please email info@sefglobal.com with your CV and a cover letter with why you want to be a facilitator for the program
- How do we partner with SEF on the program?
   - Please email info@sefglobal.com with your Request For Proposal (RFP) regarding potential partnership opportunity.

**Logo**
upspeak_logo_4_no_bg

**Additional context**
- Create a thread in [SEF Hive](https://sef.discourse.group/) before start working on this issue. 
- Contact the dev team if you need more information.

### PR DESCRIPTION
**Pull request by gajindubandara**
## Purpose
The purpose of this PR is to fix #965

## Goals
Initiate a new page for upspeak

## Approach
Initiated a new page for upspeak

### Screenshots
Screenshot from 2021-06-21 14-25-58

Screenshot from 2021-06-20 11-03-32

  
### Preview Link

https://pr-968-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: upspeak/functions.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            $(this).children("div").children("i").removeClass("fa-minus");
+            $(this).children("div").children("i").addClass("fa-plus");
+        }
+    }
+);
+
+
+
+
+
```
Comment (janethavi) [inline upspeak/functions.js]: Shall we remove the extra lines?

Comment (gajindubandara) [reply upspeak/functions.js]: Ok sure

File: upspeak/index.html [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+                        <div class="stats ml-auto">
+                            <i class="far fa-clock"></i> {{readingTime}} min read
+                        </div>
+                    </div>
+                </div>
+            </a>
+        </div>
+        {{/data}}
+    </div>
+</script>
```
Comment (Gimhan-minion) [inline upspeak/index.html]: Good job @gajindubandara! I guess we can remove this part for now, since we don't have a profile section ATM.

**Review by Gimhan-minion — CHANGES_REQUESTED** [review]

File: upspeak/index.html (hunk 2 above)
Comment (gajindubandara) [reply upspeak/index.html]: Yeah sure!

File: upspeak/index.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        <div class="modal-content">
+            <div class="pt-3 pr-3">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">×</span>
+                </button>
+            </div>
+            <div id="profile-modal-content" class="modal-body"></div>
+        </div>
+    </div>
+</div>
```
Comment (Gimhan-minion) [inline upspeak/index.html]: You can remove this too.

**Review by Gimhan-minion — APPROVED** [review]
Thank you @gajindubandara Good job!!!

**Review by janethavi — APPROVED** [review]
Great work @gajindubandara! Thank you for getting this done

**Pull request merged**